### ISSUE TITLE
How to find the exact hyperparameters (arguments passed to the script) to reproduce the results of transformer.

### ISSUE THREAD

**Issue by Saleh-Gholam-Zadeh** [cure-lab/LTSF-Linear#109]
Dear all.
Thanks for publishing and  sharing such a nice and clean code and congrats for your nice paper.

I want to reproduce the results of e.g Table 9  or Table 2 (any of them would be sufficient for me).

using the scripts inside the .scripts/EXP-LongForecasting/Linear/univariate I could only reproduce the results of NLinear which is of course the best one (has the lowest error) as reported in the table. 
however I can't reproduce the rest. E.g if I just change the model_name to Linear or DLinear the results are not the same as the table.
especially with the transformer I don't know which script can re-produce this results. e.g what should be the number of epochs? or what should be the learning rate? should I use the default ones?

can you please upload the script with the exact input parameters so that I can reproduce one of these tables 2 or 9?
Best regards,
Saleh

p.s.
background: For my own research, I need to do some further studies and therefore I need to save the input, prediction and groundtruth associated with the reported results.
